crates/rooch-rpc-api/src/jsonrpc_types/execute_tx_response.rs

```diff
@@ -80,14 +80,14 @@ impl From<KeptVMStatus> for KeptVMStatusView {
 
 #[derive(Debug, Clone, Serialize, Deserialize, JsonSchema)]
 pub struct AuthenticatorView {
-    pub coin_id: StrView<u64>,
+    pub scheme: StrView<u64>,
```

Comment:
Typo `scheme` -> `schema`

Reply:
> Typo `scheme` -> `schema`

I believe it's `scheme` before.

Reply:
scheme

Reply:
Or we rename it to authentication validator id.

Reply:
Renamed `scheme` to `auth_validator` for BuiltinAuthValidator and `auth_validator_id` for u64 ids.